centos_install_requirements.sh

```diff
@@ -1,54 +0,0 @@
-#!/usr/bin/env bash
-set -ex
-
-# shellcheck disable=SC1091
-source lib/logging.sh
-# shellcheck disable=SC1091
-source lib/common.sh
-
-sudo dnf install -y libselinux-utils
```

Comment:
I think this is missing in package [list](https://github.com/metal3-io/metal3-dev-env/blob/c6f9b23c0906a068dded9056f10b9acf5d3c25bb/vm-setup/roles/packages_installation/defaults/main.yml#L56)?

Reply:
This step is done via
```
- name: Enable SELinux
  ansible.posix.selinux:
    policy: targeted
    state: permissive
  when: ansible_selinux.status == "enabled"
```